src/Symfony/Component/Mailer/Bridge/Mailjet/Tests/Transport/MailjetApiTransportTest.php

```diff
@@ -102,15 +102,21 @@ public function testSendSuccess()
     {
         $json = json_encode([
             'Messages' => [
-                'foo' => 'bar',
+                [
+                    'Status' => 'success',
+                    'To' => [
+                        [
+                            'Email' => 'passenger1@mailjet.com',
+                            'MessageUUID' => '7c5f9f29-42ba-4959-b19c-dcd8b2f327ca',
+                            'MessageID' => 576460756513665525,
```

Comment:
yes, this test should be made to work on 32b. We decode with the  JSON_BIGINT_AS_STRING flag so I guess this should be turned into a string here